next.config.js

```diff
@@ -29,10 +28,12 @@ const nextConfig = {
          rules.forEach((rule) => {
             rule.use.forEach((moduleLoader) => {
                if (
+                  moduleLoader.options?.modules &&
```

Comment:
I'm not sure if that fixed the issue or just prevented the code execution (which was originally useful)

Reply:
Without this, the build was failing because in some cases the "modules" was undefined